backend/open_webui/routers/auths.py

```
@@ -53,7 +60,34 @@
     from ldap3 import Server, Connection, NONE, Tls
     from ldap3.utils.conv import escape_filter_chars
 
+from fastapi import FastAPI, HTTPException, Depends, Request
```

Ruff (0.12.2)

`fastapi.FastAPI` imported but unused

Remove unused import: `fastapi.FastAPI`

(F401)